### PR TITLE
Add mistral-inference

### PR DESCRIPTION
**Pull request by jeongseok-meta**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jeongseok-meta** [comment]
@conda-forge/help-python, ready for review!

**Comment by jeongseok-meta** [comment]
@ocefpaf sorry for the direct ping. Would you mind reviewing this PR when you have a moment? Thank you!

**Comment by ocefpaf** [comment]
> @ocefpaf sorry for the direct ping. Would you mind reviewing this PR when you have a moment? Thank you!

Let's wait for someone that knows about torch and cuda to chime in.

**Comment by jeongseok-meta** [comment]
Oh, `TORCH_CUDA_ARCH_LIST` is not needed (assuming it passes CI). Just removed, and now it just has pytorch run dep.

File: recipes/mistral-common/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mistral_common-{{ version }}.tar.gz
+  sha256: 110e1b937aee99fdcf3cefc6b4e924e7e22ff445246090fe9215051245caa59c
+
+build:
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/mistral-common/meta.yaml:15]:
```suggestion
  skip: true  # [osx or win]
```

File: recipes/mistral-common/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -exo pipefail
+
+python -m pip install . -vv --no-deps --no-build-isolation
```
Comment (ocefpaf) [inline recipes/mistral-common/build.sh:5]: Please remove this file and add

```
script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

to your build section.

**Comment by jeongseok-meta** [comment]
Done. Thanks for the suggestions!

**Comment by jeongseok-meta** [comment]
CIs are all green! :)

**Comment by jeongseok-meta** [comment]
@ocefpaf PTAL at your convenience. Thanks!

**Pull request merged**